docs/standard/analyzers/api-analyzer.md

```diff
@@ -0,0 +1,62 @@
+
+![](media/api-analyzer/warnings.jpg)
+
+Any warnings can be suppressed by right-clicking on the highlighted member and selecting **Quick Actions and Refactorings**. Here you get two suppression options: locally (in source) or globally (in a suppression file). We encourage developers to use global suppression to ensure consistency of API usage across their project.
+
+## Discovering cross-platform issues
+Similar to deprecated APIs the analyzer squiggles out all APIs that are not cross-platform supported (For example `Console.WindowWidth` works on Windows but not on Linux and macOS). The diagnostic ID is shown in Error List window. You can suppress warning by right click and choosing "Quick Actions and Refactorings". Unlike deprecation cases where you have two options: either keep using deprecated member and suppress warnings or not use it at all, here if you are developing your code only for certain platform you can suppress all warnings for all other  platforms you don't plan to run your code on. To do so, you just need to edit your project file and add a property `PlatformCompatIgnore` that lists all platforms to be ignored:
+```
+<PropertyGroup>
+    <PlatformCompatIgnore>Linux;MacOSX</PlatformCompatIgnore>
+</PropertyGroup>
+```
+If your code targets multiple platforms and you want to take advantage of an API not supported on some of them, you can guard that part of the code with an `if`-statement:
+
+```CSharp
+if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+{
+     var w = Console.WindowWidth;
+     // More code
+}
+```
+You can also conditionally compile per target framework/operating system, but you currently need to do that manually.
+
+## Supported diagnostics
+Right now the analyzer handles the following cases:
+* Usage of a .NET Standard API that throws `PlatformNoSupportedException` (PC001)
+* Usage of a .NET Standard API that isn't available on the .NET Framework 4.6.1 (PC002)
+* Usage of a native API that doesn’t exist in UWP (PC003)
+* Usage of an API that is marked as deprecated (DEXXXX)
+
+## CI machine
+All these diagnostics are available not only in the IDE, but also on the command line as part of building your project, which includes the CI server.
+
+## Configuration
+It is up to a user to decide how the diagnostics should be treated: as warnings, errors, suggestions, or to be turned off. For example as an architect you can decide that compatibility issues are errors, some deprecation are warnings and some are only suggestions. You can configure this separately by diagnostic ID and by project. To do so in your project tree -> Dependencies -> Analyzers -> Microsoft.DotNet.Analyzers.Compatibility, right click on the diagnostic ID and Set Rule Set Severity and pick a desired option. 
```

Comment:
comma after example and architect
some deprecation are warnings and some are only suggestions --> calls to some deprecated APIs generate warnings, while others only generate suggestions (comma after warnings)